test/e2e/framework/helpers.go

```diff
@@ -13,11 +14,61 @@ import (
 const (
 	ClusterMonitorConfigMapName      = "cluster-monitoring-config"
 	UserWorkloadMonitorConfigMapName = "user-workload-monitoring-config"
-	E2eTestLabelName                 = "app.kubernetes.io/created-by"
-	E2eTestLabelValue                = "cmo-e2e-test"
-	E2eTestLabel                     = E2eTestLabelName + ": " + E2eTestLabelValue
+
+	E2eTestLabelName  = "app.kubernetes.io/created-by"
+	E2eTestLabelValue = "cmo-e2e-test"
+	E2eTestLabel      = E2eTestLabelName + ": " + E2eTestLabelValue
 )
 
+// BuildCMOConfigMap returns a ConfigMap holding the provided Cluster
+// Monitoring Operator's config.
+// If the config isn't valid YAML, the test will fail.
+func (f *Framework) BuildCMOConfigMap(t *testing.T, config string) *v1.ConfigMap {
```

Comment:
I may have missed it, sorry if it was the case, and I know all the test do that as well, but how about a 'dummy'test that just test the creation of a minimum ConfigMap and UserWorkloadConfigMap? Totally a nit,tho.